src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/MaxLengthAttribute.cs

```diff
@@ -122,12 +122,21 @@ public static bool TryGetCount(object value, out int count)
                 return true;
             }
 
-            PropertyInfo property = value.GetType().GetRuntimeProperty("Count");
+            PropertyInfo property = null;
+            try
+            {
+                property = value.GetType().GetRuntimeProperty("Count");
+            }
+            catch (TypeAccessException)
```

Comment:
As there aren't other uses of this pattern in this library, could you please add a comment like, "// On CoreRT, this property may not be enabled for reflection."

Reply:
Done https://github.com/dotnet/corefx/pull/24101/commits/6170e275ba59ec3b90e6b0cacd9c07f0772c8394.